web/client/utils/__tests__/CatalogUtils-test.js

```diff
@@ -984,4 +984,26 @@ describe('Test the CatalogUtils', () => {
         expect(layer.provider).toBe(RECORD.provider);
         expect(layer.name).toBe(RECORD.provider);
     });
+    it('buildServiceUrl', ( ) => {
+        const WMSService = {
+            type: "wms",
+            url: "https://a.example.com/wms",
+            domainAliases: [
+                "https://b.example.com/wms",
+                "https://c.example.com/wms"
+            ]
+        };
+        const otherService = {
```

Comment:
try also with a service that has not this domainAliases array
or with a comma separated list for url

Reply:
Extended test cases.